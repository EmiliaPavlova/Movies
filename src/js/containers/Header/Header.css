.background {
  padding: 10px 80px;
  background: linear-gradient(
    rgba(0, 0, 0, 0.5), 
    rgba(0, 0, 0, 0.5)
  ), url('./assets/background.jpg');
}