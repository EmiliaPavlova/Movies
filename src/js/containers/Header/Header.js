import React, { Component, Fragment } from 'react';
import Logo from '../LogoContainer/LogoContainer';
import Search from '../SearchContainer/SearchContainer';
import './Header.css';

class Header extends Component {
  render() {
    return (
      <Fragment>
      <div className='background'>
        <Logo />
        <Search />
      </div>
      </Fragment>
    );
  }
}

export default Header;
