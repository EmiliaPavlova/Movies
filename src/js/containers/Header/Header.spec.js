import React from 'react';
import { mount } from 'enzyme';
import Header from './Header';
import './Header.css';

const mountHeader = props => mount(<Header {...props} />);

describe('Header', () => {
  describe('Snapshot', () => {
    it('matches the snapshot', () => {
      const wrapper = mountHeader();
      expect(wrapper).toMatchSnapshot();
    })
  });
});
