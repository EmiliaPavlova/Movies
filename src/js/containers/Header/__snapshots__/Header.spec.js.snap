// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot matches the snapshot 1`] = `
<Header>
  <div
    className="background"
  >
    <LogoContainer>
      <Logo
        text="netflixroulette"
      >
        <div
          className="logo"
        >
          netflixroulette
        </div>
      </Logo>
    </LogoContainer>
    <SearchContainer>
      <div
        className="searchWrapper"
      >
        <Search
          actionButton={
            <Button
              className="submitButton"
              onClick={[Function]}
              title="Search"
              type="submit"
            />
          }
          conditions={
            <div
              className="label"
            >
              Search by
              <div
                className="condition"
              >
                <Button
                  className="searchButton"
                  onClick={[Function]}
                  title="Title"
                />
              </div>
              <div
                className="condition"
              >
                <Button
                  className="searchButton"
                  onClick={[Function]}
                  title="Genre"
                />
              </div>
            </div>
          }
          title="Find your movie"
        >
          <div
            className="title"
          >
            Find your movie
          </div>
          <input />
          <div
            className="buttons"
          >
            <div
              className="label"
            >
              Search by
              <div
                className="condition"
                key="0"
              >
                <Button
                  className="searchButton"
                  onClick={[Function]}
                  title="Title"
                >
                  <button
                    className="button searchButton "
                    onClick={[Function]}
                  >
                    Title
                  </button>
                </Button>
              </div>
              <div
                className="condition"
                key="1"
              >
                <Button
                  className="searchButton"
                  onClick={[Function]}
                  title="Genre"
                >
                  <button
                    className="button searchButton "
                    onClick={[Function]}
                  >
                    Genre
                  </button>
                </Button>
              </div>
            </div>
            <span
              className="actionButton"
            >
              <Button
                className="submitButton"
                onClick={[Function]}
                title="Search"
                type="submit"
              >
                <button
                  className="button submitButton "
                  onClick={[Function]}
                  type="submit"
                >
                  Search
                </button>
              </Button>
            </span>
          </div>
        </Search>
      </div>
    </SearchContainer>
  </div>
</Header>
`;
